avocado/utils/cpu.py

```diff
@@ -473,6 +475,35 @@ def get_pid_cpus(pid):
             continue
     return list(cpus)
 
+def numa_node_has_cpus():
```

Comment:
change name to get_numa_node_with_cpus

Reply:
@abdhaleegit I woulld recommend the method name to be "get_numa_node_has_cpus", instead of  "get_numa_node_with_cpus" because the next method has similar name [ numa_nodes_with_assigned_cpus ], 
this should avoid the confuses.